### DISCUSSION TITLE
docker compose up -d时速度太慢，有什么解决方案么？

### DISCUSSION THREAD

**Discussion by duguyue8** [halo-dev/halo#4396] [Q&A]
一个小时才拉取了几百KB
![详细信息](https://pic7.58cdn.com.cn/nowater/webim/big/n_v2dabfaf13a75a4d92870afee8a1c87db6.png)

**Comment by JohnNiang** [comment]
建议配置一下镜像加速器，或者配置 Docker 代理。可参考：
- https://docs.docker.com/config/daemon/systemd/#httphttps-proxy
- https://help.aliyun.com/zh/acr/user-guide/accelerate-the-pulls-of-docker-official-images

**Reply by duguyue8** [reply]
收到，感谢
